pbs/usersync.go

```diff
@@ -157,7 +157,9 @@ func OptOut(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 
 	err := VerifyRecaptcha(rr)
 	if err != nil {
-		glog.Infof("Optout failed recaptcha: %v", err)
+		if glog.V(2) {
```

Comment:
I think we should just use the standard log levels and change the `minloglevel ` flag to 1 (warn). I don't see the need for custom levels. This assumes we have the appropriate levels used in the app, which shouldn't be hard to fix.

Reply:
such a bummer.